### ISSUE TITLE
Not converting anything?

### ISSUE THREAD

**Issue by cayhorstmann** [fabianmoronzirfas/to-markdown-cli#30]
I just installed version 0.4.1 on Ubuntu Linux 18.04. I must be doing something wrong, because not even the simplest files are converted. For example, if `test.html` is

```
<html>
  <head>
    <title>A test</title> 
  </head> 
<body>
   <h1>A test</h1>
    <p>A test</p>
</body>
</html>
```
running
```
html2md -i test.html
```
does nothing at all. Any idea what I might be doing wrong?

**Comment by fabianmoronzirfas** [comment]
Hi @cayhorstmann 

Sorry I can't reproduce the error.  With v0.3.5 I can do this

```plain
$ cat test.html
> <html>
>   <head>
>     <title>A test</title>
>   </head>
> <body>
>    <h1>A test</h1>
>     <p>A test</p>
> </body>
> </html>
$ html2md -i test.html
> A test
> 
> A test
> ======
> 
> A test
```

**Comment by fabianmoronzirfas** [comment]
Sorry. You are right the input file flag seems to be broken. Try 

```bash
cat test.html | html2md
```

**Comment by fabianmoronzirfas** [comment]
Published v0.4.2 with a fix for that.

**Comment by fabianmoronzirfas** [comment]
@all-contributors please add @cayhorstmann for bug
